src/messagingsession/PrefetchOn.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+enum PrefetchOn {
+  Connect = 'connect',
```

Comment:
Do we ever add more values to this? If so, can you put a comment here (I think David put a comment to do this in 2.x branch).

Reply:
Yes, so I originally put it as a boolean. David mentioned we could expand on this later, for example add in "membership".

Reply:
Yeah but i mean can you explain why it is an enum instead of boolean like David said. Otherwise, whoever modify the code next to you would not understand why this has to be an enum :D

Reply:
We decided to use enum instead of boolean since we could further expand this feature to prefetch on membership changes etc. So want to make this easy to expand.